### PR TITLE
calendar was added

### PR DESCRIPTION
**Pull request by Yuliia-Hrinchenko1**
- [DEMO LINK](https://Yuliia-Hrinchenko1.github.io/layout_calendar/)
- [TEST REPORT LINK](https://Yuliia-Hrinchenko1.github.io/layout_calendar/report/html_report/)

### CODE REVIEW

**Review by kuzmi491 — APPROVED** [review]
I think it's great=)

File: src/index.html [hunk 1]
```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <main class="main">
+      <div class="calendar calendar--month-length-29 calendar--start-day-sun">
```
Comment (sergii-nosachenko) [inline src/index.html]: Checklist #2

File: src/styles/blocks/calendar.scss [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    font-size: 30px;
+    text-align: center;
+
+    border: 1px solid $c-black;
+    background-color: $c-grey;
+
+    transition-duration: 0.5s;
+
+    &:hover {
+      cursor: pointer;
```
Comment (sergii-nosachenko) [inline src/styles/blocks/calendar.scss]: Better set cursor in element, not in pseudo-class

File: src/styles/blocks/calendar.scss [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    align-items: center;
+    @include square($cell-size);
+
+    font-size: 30px;
+    text-align: center;
+
+    border: 1px solid $c-black;
+    background-color: $c-grey;
+
+    transition-duration: 0.5s;
```
Comment (sergii-nosachenko) [inline src/styles/blocks/calendar.scss]: Add transition properties as a list.

File: src/styles/utils/_functions.scss [hunk 4]
```diff
@@ -0,0 +1,7 @@
+@function calendarWidth ($celldWidth, $gaps, $daysInRow, $paddings) {
+  @return ($daysInRow * $celldWidth + $gaps * ($daysInRow - 1) + 2 * $paddings);
```
Comment (sergii-nosachenko) [inline src/styles/utils/_functions.scss]:
```suggestion
  @return $daysInRow * $celldWidth + $gaps * ($daysInRow - 1) + 2 * $paddings;
```

File: src/styles/blocks/calendar.scss [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+
+    &:hover {
+      cursor: pointer;
+      background-color: #efdce6;
```
Comment (sergii-nosachenko) [inline src/styles/blocks/calendar.scss]: Add color to variables

**Review by sergii-nosachenko — CHANGES_REQUESTED** [review]

**Review by sergii-nosachenko — APPROVED** [review]
Good work 👍

**Pull request closed without merging**